src/class-billmate-common.php

```diff
@@ -104,7 +104,7 @@ public function get_address_fields()
                     </p>
                     <div class="clear"></div>
                     <p class="form-row form-row-first">
-                        <input type="text" autocomplete="off" name="pno" label="12345678-1235" class="form-row-wide input-text" value="<?php echo isset($_SESSION['billmate_pno']) ? $_SESSION['billmate_pno'] : ''; ?>"/>
+                        <input type="text" autocomplete="off" name="pno" label="12345678-1235" class="form-row-wide input-text" value="<?php echo WC()->session->get('billmate_pno') !== null ? C()->session->get('billmate_pno') : ''; ?>"/>
```

Comment:
C()->session->get('billmate_pno')
Should be
WC()->session->get('billmate_pno')